neuraxle/base.py

```diff
@@ -799,6 +800,15 @@ def __reversed__(self) -> 'BaseStep':
         """
         return self.reverse()
 
+    def __repr__(self):
+        output = 'step name : ' + self.name + '\n' \
+                 + 'hyperparameters : ' + pprint.pformat(self.get_hyperparams())
```

Comment:
I would prefer something like this: 

```
output = self.__class__.__name__ + "(\n\tname=" + self.name + "\n\t, hyperparameters=" + pprint.pformat(self.hyperparams)+"\n)"
```

note: here it's important to just use self.hyperparams and not the getter which might be recursive.

Reply:
done